app/views/non_hmis_uploads/index.haml

```diff
@@ -28,11 +28,9 @@
               %td= upload.created_at
               %td
                 = upload.import_time
-                -# TODO
-                  - if upload.import_log.present?
-                    = link_to upload.import_time, import_path(id: upload.import_log.id)
-                  - else
-                    = upload.import_time
+                - if upload.import_log_id.present?
+                  = link_to upload.import_time, import_path(id: upload.import_log_id)
+                - else
+                  = upload.import_time
```

Comment:
Flagging for @eanders to review in the UI, not sure why this is changing

Reply:
I just had some local data missing import_log_id (but with import_time set. Looking briefly at the git history i got the impression this used to be possible

Reply:
I think we're good now.  I just took it out for now.